openjfx/src/test/java/science/aist/imaging/service/openjfx/imageprocessing/wrapper/WriteableImageFactoryTest.java

```diff
@@ -0,0 +1,87 @@
+package science.aist.imaging.service.openjfx.imageprocessing.wrapper;
+
+import javafx.scene.image.Image;
+import javafx.scene.image.PixelWriter;
+import javafx.scene.image.WritableImage;
+import javafx.scene.paint.Color;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import science.aist.imaging.api.domain.wrapper.ImageFactory;
+import science.aist.imaging.api.domain.wrapper.ImageWrapper;
+import science.aist.imaging.api.domain.wrapper.implementation.ImageFactoryFactory;
+
+import javax.imageio.ImageIO;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Random;
+
+/**
+ * <p>Created by Christoph Praschl  on 16.06.2021</p>
+ * <p>Test class for {@link WriteableImageFactory}</p>
+ *
+ * @author Christoph Praschl christoph.praschl@fh-hagenberg.at
```

Comment:
Same - although in tests you do not need to add the `since` annotation in my opinion.